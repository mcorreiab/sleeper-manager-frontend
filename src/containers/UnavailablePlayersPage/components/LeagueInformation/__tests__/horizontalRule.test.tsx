import renderer from "react-test-renderer";
import HorizontalRule from "../horizontalRule";

it("should render horizontal rule without props", () => {
  expect(renderer.create(<HorizontalRule />).toJSON()).toMatchSnapshot();
});
