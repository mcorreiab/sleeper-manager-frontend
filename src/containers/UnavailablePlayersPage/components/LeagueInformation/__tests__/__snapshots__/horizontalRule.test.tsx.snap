// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render horizontal rule without props 1`] = `
<hr
  className="border border-t-0 border-[#ffffff26] border-solid"
/>
`;
